pyproject.toml

```diff
@@ -56,7 +56,7 @@ momepy = ["datasets/bubenec.gpkg", "datasets/tests.gpkg"]
 [tool.ruff]
 line-length = 88
 select = ["E", "F", "W", "I", "UP", "N", "B", "A", "C4", "SIM", "ARG"]
-ignore = ["B006", "F403"]
+ignore = ["B006", "F403", "SIM108"]
```

Comment:
This is to avoid converting

```py
    if weighted:
        r = graph.cardinalities / geometry.length
    else:
        r = graph.cardinalities
```
into an inline. I find this way more readable.